src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -169,7 +169,7 @@ const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleC
         choices.length === 0 ? (<NoOptions className={styles['no-options']} />) : (
           <Listbox
             className={styles.dropdown}
-            ariaExpanded={true}
+            expanded={true}
```

Comment:
Oh, I just realized this is always expanded. I now wonder if this has any meaning to the a11y tree since it is always `true`. 

But at least it is part of the Listbox API!

Reply:
I think we might not be using the `aria-expanded` attribute correctly here.

https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

The `aria-expanded` is used on the element that has a popup. The `aria-expanded` indicates to the accessibility tree that the popup is either expanded (visible) or collapsed (not visible)_.